.changeset/moody-kids-double.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-cloud-sdk/http-client': minor
+'@sap-cloud-sdk/resilience': minor
+---
+
+[New Functionality] The HTTP request config is not part of the middleware context so that middlewares can change the request config easily.
```

Comment:
[nth] I don't think that I understand this sentence. Maybe this could be rephrased to be clearer.